pyaedt/workflows/installer/pyaedt_installer.py

```diff
@@ -22,55 +22,77 @@
 
 """Methods to add PyAEDT in AEDT."""
 
+import logging
 import os
+import shutil
 
 from pyaedt.generic.general_methods import read_toml
 from pyaedt.workflows import customize_automation_tab
+import pyaedt.workflows.templates
 
 
 def add_pyaedt_to_aedt(
     aedt_version,
-    personallib,
+    personal_lib,
 ):
     """Add PyAEDT tabs in AEDT.
 
     Parameters
     ----------
     aedt_version : str
         AEDT release.
-    personallib : str
+    personal_lib : str
         AEDT Personal Lib folder.
```

Comment:
```suggestion
        AEDT personal library folder.
```
Descriptions should use plain English.